lib/byron/test/integration/Cardano/Wallet/Byron/Faucet.hs

```diff
@@ -0,0 +1,21 @@
+{-# LANGUAGE DataKinds #-}
+
+module Cardano.Wallet.Byron.Faucet
+    ( Faucet(..)
+
+    , initFaucet
+    ) where
+
+import Prelude
+
+import Control.Concurrent.MVar
+    ( newMVar )
+import Test.Integration.Faucet
+    ( Faucet (..), icaMnemonics, rndMnemonics, seqMnemonics )
+
+initFaucet :: IO Faucet
+initFaucet = Faucet
+    <$> newMVar seqMnemonics
+    <*> newMVar icaMnemonics
+    <*> newMVar rndMnemonics
+    <*> newMVar [] -- FIXME: txBuilder for external transaction.
```

Comment:
here, if I understand differences between cardano-node and jormungadr, we will definitely NOT reuse `mkTxBuilder` from Jormungandr.Faucet but deliver dedicated one

Reply:
>  reuse mkTxBuilder from Jormungandr.Faucet but deliver dedicated one. Right?

Precesily. The tx builder in Jormungandr extensively uses jcli and construct transactions that follow Jormungandr's binary format. For cardano-node, it's a different story.